prod.Dockerfile

```diff
@@ -15,73 +15,122 @@ COPY . .
 RUN pnpm install -f --offline
 
 
-FROM base_builder as backend
-RUN apk add caddy
+FROM base_builder AS backend_builder
 WORKDIR /usr/src/app/packages/hoppscotch-backend
 RUN pnpm exec prisma generate
 RUN pnpm run build
-COPY --from=base_builder /usr/src/app/packages/hoppscotch-backend/backend.Caddyfile /etc/caddy/backend.Caddyfile
+RUN pnpm --filter=hoppscotch-backend deploy /dist/backend --prod
+WORKDIR /dist/backend
+RUN pnpm exec prisma generate
+
+FROM node:20-alpine3.19 AS backend
+RUN apk add caddy
+RUN npm install -g pnpm
+
+RUN addgroup -S hoppgroup && adduser -S hoppuser -G hoppgroup
+
+COPY --from=base_builder  /usr/src/app/packages/hoppscotch-backend/backend.Caddyfile /etc/caddy/backend.Caddyfile
+COPY --from=backend_builder --chown=hoppuser:hoppgroup --chmod=755 /dist/backend /dist/backend
+COPY --from=base_builder --chown=hoppuser:hoppgroup --chmod=755 /usr/src/app/packages/hoppscotch-backend/prod_run.mjs /dist/backend
+
 # Remove the env file to avoid backend copying it in and using it
-RUN rm "../../.env"
 ENV PRODUCTION="true"
 ENV PORT=8080
 ENV APP_PORT=${PORT}
 ENV DB_URL=${DATABASE_URL}
-CMD ["node", "/usr/src/app/packages/hoppscotch-backend/prod_run.mjs"]
+
+USER hoppuser
+
+WORKDIR /dist/backend
+
+CMD ["node", "prod_run.mjs"]
 EXPOSE 80
 EXPOSE 3170
 
-FROM base_builder as fe_builder
+FROM base_builder AS fe_builder
```

Comment:
![image](https://github.com/user-attachments/assets/3906aa73-dc14-4b7c-bc66-af2241b8fe51)

I get this error when I try running the app container standalone.
Context: Subpath is false.

Reply:
should be fixed now.